addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -146,3 +146,31 @@
 .icon_rotates {
     transform: rotate(180deg);
 }
+
+/* Style of the tiles allowing the user to load a sample survey. */
+.o_survey_sample_tile {
+    position: relative;
+    width: 150px;
+    height: 150px;
+    img {
+        max-height: 125px;
+    }
+    div {
```

Comment:
Targeting "img" and "div" tags is usually a bit "unsafe".
Since those rules would also apply if someone modifies your template to add some more divs for example.
Could you give those some specific classes (e.g: "o_survey_sample_tile_image")?